src/selectors/profile.js

```diff
@@ -864,6 +864,23 @@ export const getThreadIdToNameMap: Selector<Map<Tid, string>> = createSelector(
   }
 );
 
+export const getProcessIdToNameMap: Selector<Map<Pid, string>> = createSelector(
+  getThreads,
+  (threads) => {
+    const processIdToNameMap = new Map();
+    for (const thread of threads) {
+      if (thread.name !== 'GeckoMain' || !thread.pid) {
```

Comment:
Should we use that instead?
```suggestion
      if (thread.isMainThread || !thread.pid) {
```